src/Flicking.ts

```diff
@@ -35,14 +37,12 @@ export type FlickingEvents = {
   [EVENTS.NEED_PANEL]: NeedPanelEvent;
   [EVENTS.VISIBLE_CHANGE]: VisibleChangeEvent;
   [EVENTS.REACH_EDGE]: ReachEdgeEvent;
-  [EVENTS.CONTENT_ERROR]: void;
-};
+}
 
 /**
  * @param element A base element for the eg.Flicking module. When specifying a value as a `string` type, you must specify a css selector string to select the element.<ko>eg.Flicking 모듈을 사용할 기준 요소. `string`타입으로 값 지정시 요소를 선택하기 위한 css 선택자 문자열을 지정해야 한다.</ko>
```

Comment:
Does the interface look right in the doc documentation?

`@param` is used in methods, constructors, and events.
In the case of interface, `@typedef` is required and `@property` is used.